### ISSUE TITLE
No option to add integration from HA UI

### ISSUE THREAD

**Issue by JeremyKennedy** [dalinicus/homeassistant-aerogarden#29]
**Describe the bug**
I just installed this integration through HACS's default repository, restarted HA, and tried adding the Aerogarden integration, but it's not presented as an option in the UI. I had an old custom Aerogarden integration installed that I used to maintain, but it stopped working a while back and now I'm trying out yours. I've removed the old one, but still can't see yours in the integration list. Any ideas?

**To Reproduce**
1. Install from HACS
2. Restart HA
3. Under Integrations, click Add Integration
4. Search for Aerogarden, no results

**Comment by dalinicus** [comment]
When rebooting home assistant, do you have any errors pop up in your logs?  It sounds like the integration isn't loading properly, possibly due to a data conflict between your old integration.

**Comment by JeremyKennedy** [comment]
I checked the logs, I noticed it wasn't even being mentioned in the "We found a custom integration XX which has not been tested by Home Assistant." section. Noticed there was an update for the Aerogarden integration, updated it, rebooted HA, checked the logs again, and I see it getting loaded now. Based on the changes in the release, there's no way that the release fixed it. I think it was just HACS installing it properly this time. I feel like the solution would have been to remove, then re-add the integration. Anyway, all good now, it works! Thank you.

Though I now see that there is no option to control the light... That was something I could do in my integration, though it was pretty buggy due to the 3-step toggle behavior.
